ready-to-use fields/SEO/export.json

```diff
@@ -0,0 +1,84 @@
+[
```

Comment:
Can we change the folder path to not have any spaces, and instead of `SEO/export.json` perhaps the file can be called `seo.json`? So the final result might be:

- `field-library/README.md`
- `field-library/seo.json`
- `field-library/xyz.json`
- etc.

Reply:
Done.